core/src/main/scala/sttp/tapir/EndpointIO.scala

```diff
@@ -489,6 +490,7 @@ object EndpointIO {
       description: Option[String],
       examples: List[Example[T]],
       deprecated: Boolean,
+      hideInDocs: Boolean,
```

Comment:
Since we already have this data in the `Schema`, I think we don't need it in `Info`? Otherwise we are duplicating information

Reply:
that's because in OpenAPI, you can deprecate either an input parameter (such as a query param or header), or you can deprecate the schema - the `deprecated` attribute can be present in both places. I don't think we need such a distinction here.

Reply:
fixed